include/mpd/async.h

```diff
@@ -209,6 +209,10 @@ mpd_malloc
 char *
 mpd_async_recv_line(struct mpd_async *async);
 
+
+struct mpd_binary
+mpd_async_recv_binary(struct mpd_async *async, const unsigned binary);
```

Comment:
This `const` has no effect.
It is not obvious what the parameter `binary` means.
Note that this method is called "recv" but it never receives anything.

Reply:
Changed.